src/app/contribute/guidelines/component-guidelines/index.html

```diff
@@ -0,0 +1,269 @@
+    </p>
+
+  <stache-page-anchor>
+    File an issue
+  </stache-page-anchor>
+
+  <p>
+    Before you start any work, file an issue</a>. The sooner we can discuss potential code changes, the better. We also use the issue to track and assign any work needed. To file a issue with SKY UX, follow these instructions:
+  </p>
+
+  <ol>
+    <li>
+      Create an issue in the <a stacheRouterLink="/contribute/packages">applicable repo</a>.
+    </li>
+    <li>
+      Delete the default fields for <a stacheRouterLink="." fragment="report-bugs">reporting bugs</a> and describe the contribution and potential code changes to SKY UX.
+    </li>
+    <li>
+      Subscribe to the issue to receive notifications.
+    </li>
+  </ol>
+
+  <stache-page-anchor>
+    Get the code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      <a stacheRouterLink="https://help.github.com/articles/fork-a-repo/">Fork the master branch</a> into your own GitHub repo.
+    </li>
+    <li>
+      Create a branch named after the feature you will contribute. The branch name should be kebab-case (my-new-feature).
+    </li>
+    <li>
+      Clone your repo locally and run <stache-code>npm install</stache-code> from your local repo's directory to install all required dependencies.
+    </li>
+    <li>
+      <a stacheRouterLink="." fragment="write-visual-regression-tests"></a>Run your first round of visual regression tests</a>. This establishes baseline screenshots of the components in the master repo so that as you make changes and run your visual regression tests, your changes are validated against the state of the repo before the changes.
+    </li>
+  </ol>
+
+  <stache-page-anchor>
+    Write code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      Launch a command prompt, <stache-code>cd</stache-code> to the folder where you cloned your branch, and run <stache-code>npm run watch</stache-code>.
+    </li>
+    <li>
+      Launch a second command prompt, <stache-code>cd</stache-code> to the folder where you cloned your branch, and run <stache-code>skyux serve</stache-code>. This launches the SKY UX component demos page where you write the demo components to test your code.
+    </li>
+    <ul>
+      <li>
+        SKY UX demos are authored with <a stacheRouterLink="https://github.com/blackbaud/skyux-cli">SKY UX Builder</a>. If this is your first time using SKY UX Builder, you must install the CLI as a global NPM package before you run <stache-code>skyux serve</stache-code>:
+      </li>
+      <stache-code>$ npm install @blackbaud/skyux-cli -g</stache-code>
```

Comment:
I would change `$ npm install @blackbaud/skyux-cli -g` to `npm i -g @skyux-sdk/cli` (No dollar sign and updated package).